### DISCUSSION TITLE
A confusing widget on "AAT" 's UI

### DISCUSSION THREAD

**Discussion by b450mhdv** [bailuk/AAT#145] [General]
Hello, I enjoy using this app, it is very useful for me! However, I found a misleading widget in this app.
I think the "off" widget is only to show the status at the first glance. I think it confuses users as there are few similar widgets in popular apps. Since it's very similar to a text box, and it's not similar to a "button" or "switch" in popular apps, users might think that the widget is just for showing status.
So, just as a personal suggestion, could you try to redesign the widget? And thanks for your useful app!

<img src="https://user-images.githubusercontent.com/78015408/171568349-c24ca3e9-a97c-42c0-92d4-7a1382045856.jpg" width="300px"/>

**Comment by ANDREW007CZ** [comment]
It does not cunfuse me. It's simple and great I would say. Once you know how it works, you just know.
